test/test_xpu.py

```diff
@@ -0,0 +1,43 @@
+# Owner(s): ["module: intel"]
+
+import sys
+import unittest
+
+import torch
+from torch.testing._internal.common_utils import NoTest, run_tests, TEST_XPU, TestCase
+
+if not TEST_XPU:
+    print("XPU not available, skipping tests", file=sys.stderr)
+    TestCase = NoTest  # noqa: F811
+
+TEST_MULTIXPU = torch.xpu.device_count() > 1
+
+
+class TestXpu(TestCase):
+    def test_device_behavior(self):
+        current_device = torch.xpu.current_device()
+        torch.xpu.set_device(current_device)
+        self.assertEqual(current_device, torch.xpu.current_device())
+
+    @unittest.skipIf(not TEST_MULTIXPU, "only one GPU detected")
+    def test_multi_device_behavior(self):
+        current_device = torch.xpu.current_device()
+        target_device = (current_device + 1) % torch.xpu.device_count()
+
+        with torch.xpu.device(target_device):
+            self.assertEqual(target_device, torch.xpu.current_device())
+        self.assertEqual(current_device, torch.xpu.current_device())
+
+        with torch.xpu._DeviceGuard(target_device):
+            self.assertEqual(target_device, torch.xpu.current_device())
+        self.assertEqual(current_device, torch.xpu.current_device())
+
+    def test_get_device_name(self):
```

Comment:
Looks like following methods are not covered by this test. Is there a possibility to include these tests ?
```
device_of
get_device_capability
get_device_properties
```

Reply:
I refine UT to cover `get_device_capability` and `get_device_properties`. Due to the lack of `Allocator` runtime, we can NOT cover `device_of`. But we cover `torch.xpu.device`, a parent class of `device_of`. In my opinion, it is enough.